input/circuit.circom

```diff
@@ -49,7 +49,7 @@ template ArgMax (n) {
     out <== amaxs[n];
 }
 
-component main = ArgMax(1);
+component main = ArgMax(2);
```

Comment:
I needed to change this to make the example circuit work with boolify.

The reason `1` doesn't work with boolify is that the resulting circuit doesn't depend on any inputs. This is a problem for boolify because it avoids the need for any explicit constant encoding by using XOR(x, x) of an input bit to create `false`, and negating that to produce `true`. If there aren't any inputs, it can't do this.